src/utils/strings.test.js

```diff
@@ -34,4 +34,20 @@ describe('strings', () => {
       'http://localhost:8080/share/v/p/index.ifc#c:-26.91,28.84,112.47,0,16.21,-3.48',
     ])
   })
+
+
+  it('isNumeric recognizes common cases', () => {
```

Comment:
While there's nothing listed in the style guide for this, I'm not a fan of functions have have a lot of linter exclusions.

To put a number on it, this has 1 exclusion for every 2 lines of code. I would argue that this high ratio warrants that this file should not be included in compilation.

Reply:
Disagree.  This was defensive for being called with numbers when strings are expected.

But!  Turns out we only use it once, so will turn it into an exception and not test the numeric arg.

> 

yep, didn't realize it was that big.  SVG is more useful too and 10x smaller.  Removed from checkin